Podman System Reset --run-root --graph-root (#12770)
containers/podman: docs/source/markdown/podman-system-reset.1.md

**podman system reset** [*options*]

## DESCRIPTION

> **podman system reset** removes all pods, containers, images and volumes. can also be used to instantiate new directories for storing images and information about Podmans's state.

Reviewer: ```suggestion
**podman system reset** removes all pods, containers, images and volumes. Can also be used to instantiate new directories for storing images and information about Podmans's state.
```

> **podman system reset** removes all pods, containers, images and volumes. can also be used to instantiate new directories for storing images and information about Podmans's state.

Reviewer: ```suggestion
**podman system reset** removes all pods, containers, images and volumes. Can also be used to instantiate new directories for storing images and information about Podmans's state.
```

Print usage statement

#### **--run-root**

> Establishes a new run root directory and storage.conf file once the system is reset, cannot be used in conjunction with --user

Reviewer 2: ```suggestion
Establishes a new run root directory and storage.conf file once the system is reset, cannot be used in conjunction with --user.
```

#### **--graph-root**

> Establishes a new graph root directory and sotrage.conf file once the system is reset, cannot be used in conjunction with --user

Reviewer 2: ```suggestion
Establishes a new graph root directory and storage.conf file once the system is reset, cannot be used in conjunction with --user
```

#### **--user**

> Establishes new run and graph root directories and storage.conf file once the system is reset. the run and graph root directories are generated by retrieving rhe user's home and run directory.

Reviewer 2: ```suggestion
Establishes new run and graph root directories and storage.conf file once the system is reset. The run and graph root directories are generated by retrieving the user's home and run directory.
```

## EXAMPLES

> Reset the system and create a new storage.conf enerated from the given user's defaults

Reviewer 2: ```suggestion
Reset the system and create a new storage.conf file generated from the given user's defaults
```

```
$ podman system reset --user=1000:1000
```

> Reset the system and create a new storage.conf with the given run and graph root

Reviewer 2: ```suggestion
Reset the system and create a new storage.conf file with the given run and graph root.
```
